build.gradle

```diff
@@ -0,0 +1,69 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (C) 2008-2016 VoltDB Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+plugins {
+  id 'java'
+  id 'eclipse'
+  id 'com.github.johnrengelman.shadow' version '1.2.3'
+}
+
+description = 'VoltDB Kinesis Firehose export conduit'
+version='1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+configurations {
+    voltlib
+    export
+}
+
+def volt = fileTree(dir: "${voltdbhome}/voltdb", include: 'voltdb-*.jar')
+
+dependencies {
+    voltlib volt
+    voltlib ('commons-logging:commons-logging:1.1.3') { transitive = true }
+    voltlib ('commons-codec:commons-codec:1.6') { transitive = true }
+    voltlib ('commons-cli:commons-cli:1.2') { transitive = true }
+
+    export ('com.amazonaws:aws-java-sdk-kinesis:1.10.69') { transitive = true }
+    export ('com.amazonaws:aws-java-sdk-core:1.10.69') { transitive = true }
+    export ('com.fasterxml.jackson.core:jackson-databind:2.5.3') { transitive = true }
+    export ('com.fasterxml.jackson.core:jackson-annotations:2.5.0') { transitive = true }
```

Comment:
aws-java-sdk-x.x.x includes everything in both aws-java-sdk-kinesis-x.x.x.jar and aws-java-sdk-core-x.x.x.jar.  Would it be better to use aws-java-sdk-x.x.x.jar?  aws-java-sdk-1.10.72 is used in the importer.